src/styles/card.scss

```
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  margin: auto;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background-color: $white;
+  justify-content: center;
+
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    text-align: left;
+    padding-top: 40px;
```

change padding for margin for other cases where you need to add outer indent between elements
```suggestion
    margin-top: 40px;
```